### PR TITLE
Add LIP "Introduce Fee module"

### PR DESCRIPTION
**Pull request by MaximeGagnebin**

### CODE REVIEW

**Review by ManuGowda — APPROVED** [review]

**Review by IkerAlus — APPROVED** [review]
Looks all good.

File: proposals/lip-Introduce_Fee_module.md [hunk 1]
```diff
@@ -0,0 +1,161 @@
...
+* `extraFee(MODULE_ID_DPOS, COMMAND_ID_DELEGATE_REGISTRATION) = 1000000000`,
+* `extraFee(MODULE_ID_INTEROPERABILITY, COMMAND_ID_SIDECHAIN_REG) = 1000000000`.
+
+The constants `MODULE_ID_DPOS` and `COMMAND_ID_DELEGATE_REGISTRATION` are defined in [LIP "Define state and state transitions of DPoS module"][DPoS-LIP]. 
+The constants `MODULE_ID_INTEROPERABILITY` and `COMMAND_ID_SIDECHAIN_REG` are defined in [LIP "Introduce Interoperability module"][base-interoperability-LIP].
+
+
+### Fee Module Store
+
+The fee module does not store information in the state.
```
Comment (ricott1) [inline proposals/lip-Introduce_Fee_module.md:84]:
```suggestion
The Fee module does not store information in the state.
```

**Review by ricott1 — CHANGES_REQUESTED** [review]
small last typo, the rest LGTM

**Review by ricott1 — APPROVED** [review]
LGTM

**Review by mitsuaki-u — APPROVED** [review]

**Review by karmacoma — APPROVED** [review]

**Pull request merged**